src/compiler/crystal/codegen/primitives.cr

```diff
@@ -912,4 +914,84 @@ class Crystal::CodeGenVisitor
 
     node.value
   end
+
+  def void_ptr_type_descriptor
```

Comment:
I'm glad code snippets end up been useful :-). Thanks for tidying them up a bit.